### PR TITLE
[SDK-223] Exception Detail Model updated.

### PR DESCRIPTION
**Pull request by ZahidZafar**

### CODE REVIEW

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
First of all, it was requested that it should be a function. Second, all those fields should be set inside of that function and none of them outside. Third, it does not add all the possible fields that do not require native level calls.

File: Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs [hunk 1]
```diff
@@ -17,6 +20,7 @@ internal CrashReportsCountlyService(CountlyConfiguration configModel, RequestCou
         {
             _configModel = configModel;
             _requestCountlyHelper = requestCountlyHelper;
+            _startTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs:23]: set this as a static field on the lines that declares this variable

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

File: Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs (hunk 1 above)
Comment (ZahidZafar) [reply Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs:23]: Only one instance of CrashReportsCountlyService is created in our sdk. Do we need to make it static.

Comment (ArtursKadikis) [reply Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs:23]: You still didn't resolve this. Move the setting of this value to it's decleration.

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Do some more server side testing

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**